otter/convergence/steps.py

```diff
@@ -558,3 +558,17 @@ def _rcv3_check_bulk_delete(attempted_pairs, result):
         return next_step.as_effect()
     else:
         return StepResult.SUCCESS, []
+
+
+@implementer(IStep)
+@attributes([Attribute('seconds', instance_of=float)])
```

Comment:
I thought we decided against having a time encoded in the step? Given that we don't even have a mechanism to communicate this time back to any other code I think we should just leave it out.